### ISSUE TITLE
Bad combination of Bhaiksuki repha and vowel sign i/ī

### ISSUE THREAD

**Issue by dscorbett** [googlei18n/noto-fonts#1375]
### Font
NotoSansBhaiksuki-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlei18n/noto-fonts/blob/64f57055a31ee72789437d786e969ded9a1231f9/hinted/NotoSansBhaiksuki-Regular.ttf
Date: 2018-11-11

### Font version
Version 2.000;GOOG;noto-source:20181019:f8f3770

### Issue
In a Bhaiksuki cluster with a repha and the vowel sign <i>i</i> or <i>ī</i>, the hollow triangle of the vowel sign should be written in place of the solid triangle of the repha. In this font, the vowel sign is shaped relative to the base as if the repha weren’t there, and the repha hovers awkwardly to the side.

### Character data
𑰨𑰿𑰎𑰰𑰨𑰿𑰎𑰱
U+11C28 BHAIKSUKI LETTER RA
U+11C3F BHAIKSUKI SIGN VIRAMA
U+11C0E BHAIKSUKI LETTER KA
U+11C30 BHAIKSUKI VOWEL SIGN I
U+11C28 BHAIKSUKI LETTER RA
U+11C3F BHAIKSUKI SIGN VIRAMA
U+11C0E BHAIKSUKI LETTER KA
U+11C31 BHAIKSUKI VOWEL SIGN II

### Screenshot
![𑰨𑰿𑰎𑰰𑰨𑰿𑰎𑰱](https://user-images.githubusercontent.com/1124347/48315843-69ad5100-e5a9-11e8-9fac-0532d207f976.png)

**Comment by punchcutter** [comment]
Are these real combinations found in Bhaiksuki text? Noto Bhaiksuki covers all conjuncts shown in The Bhaikṣukī Manuscript of the Candrālaṃkāra, but it doesn't attempt to cover every possible theoretical conjunct.

**Comment by dscorbett** [comment]
I don’t know whether these sample syllables are attested in Bhaiksuki. There do exist Sanskrit words with <i>rki</i>, so it is plausible.

**Comment by simoncozens** [comment]
We are currently (2.001) seeing this for the input text you gave:
`[Ki=0+575|Kii=2+595|Repha=2@-320,30+0]`
![shape](https://user-images.githubusercontent.com/106728/163781985-00092f15-bce1-4150-b504-6cf292539916.png)

Does that resolve the issue? If not, how should the sequence appear?

**Comment by punchcutter** [comment]
That's not correct, but the best solution takes a lot of work. When I took over this font it was already done in a way that's not ideal. A better way would be to build it so we can compose such sequences. Unfortunately the whole thing was made originally with most combinations precomposed so if you run into a sequence that doesn't have a precomposed glyph it might not work out so well. One of the only people in the world who knows anything about Bhaiksuki doesn't like this font at all anyway so an even better solution would be to redesign the whole thing and construct it correctly from the start.

**Comment by punchcutter** [comment]
As far as how it should look...based on other real examples the repha should go where ī is and then the vowel moves up and sits on top. When I had studied enough to understand we should construct the font in a way to handle arbitrary sequences like this it was kind of too late to redo the whole thing. So right now the attested glyphs are made precomposed and anything else we get something like this where it's just a blobby mess of stuff overlapping.

**Comment by simoncozens** [comment]
Thanks for the input. That's... unfortunate. Now I look at the sources I see the problem; ideally all the vowels should be cut off and dynamically composed. This will be a lot of work.
